components/molecules/HighlightCard/highlight-card.tsx

```diff
@@ -154,7 +160,17 @@ const HighlightCard: React.FC<HighlightCardProps> = ({
               {getIcon(icon)}
             </div>
             {/* Label: Text */}
-            <div className="text-sm text-slate-600   leading-none">{label ? label : "Label"}</div>
+            <div className={clsx("text-sm text-slate-600 leading-none")}>
+              {tooltip && lable ? (
```

Comment:
typo is causing the build failure

```suggestion
              {tooltip && label ? (
```